Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/manager_app/view/child_list/ChildListActivity.java

```diff
@@ -70,6 +103,14 @@ private Long addChild(String firstName, String lastName) {
         }
     }
 
+    private void setActiveChild(){
+        List<Child> childList = childRepository.getAll();
+        for (Child child : childList)
+            if(child.getIsActive())
+                child.setIsActive(false);
+        childListAdapter.getChild(selectedChildPosition).setIsActive(true);
```

Comment:
When I tested it the active value was not setting, you could select the row but it does not look that the value is stored anywhere. I don't think we update child in db and we should update the record after selected it with is_active to true. In my opinion it should also change the color of the row, when the list is displayed on opening child list view. One child should always be active and it should be visible on the list. Could you please verify it?